domdiv/main.py

```diff
@@ -525,6 +540,14 @@ def get_resource_stream(path):
     return codecs.EncodedFile(pkg_resources.resource_stream('domdiv', path), "utf-8")
 
 
+def find_index_in_list(lst, key, value):
+    # Returns the index of the first dictonary in lst that has key with value.  Otherwise returns None
+    for i, d in enumerate(lst):
```

Comment:
This appears to not be looking a dictionaries, but at objects' attributes, contrary to the comment. It should also have a more specific name that reflects this, like `find_attribute_value` or similar. 

This could also be written inline or implemented as as `next((i for i, o in enumerate(lst) if getattr(o, key, None) == value), None)` though I'm not sure that's clearer.

Reply:
Sure, `find_index_of_object` sounds good.